### DISCUSSION TITLE
ConfyUI facelift

### DISCUSSION THREAD

**Discussion by shakuro-design** [comfyanonymous/ComfyUI#3632] [Ideas]
Hey, everyone!  My name is Kirill, I am art-director at @shakurocom agency and we use ComfyUI for work. Really love how flexible it is with Stable Diffusion.

Some time ago me and my team though it’d be cool to play around with the interface concept for ComfyUI. We updated the general look, reduced visual noise, put together a design system without affecting the current logic.

The design system includes:
1. Typography, iconography, and utility elements.
2. Node groups, elements, headings, connectors, field elements, and states.
3. Sidebar components.
4. Grid system rules.

Check it out and if you dig it, please, vote for the idea. Also, leave a comment for me to know whether the community appreciates this type of work and if we should keep going. ✌️️

![load to dafault](https://github.com/comfyanonymous/ComfyUI/assets/127851058/75d3e86a-cb24-4b13-aea8-5c19d20617c5)
![result](https://github.com/comfyanonymous/ComfyUI/assets/127851058/c81ac616-12b0-4bbc-85cd-9088ad49f408)
![queue prompt](https://github.com/comfyanonymous/ComfyUI/assets/127851058/ba267018-b468-4a32-9baf-7eeabb5d705a)

**Comment by shakuro-design** [comment]
Here is figma link: https://www.figma.com/design/npLvEHRmMjpI44gNAKc0XM/ComfyUI---FaceLift?node-id=0-1&t=CHXeRZVisGFkNqJF-1

**Comment by Thehopes** [comment]
# I like it !!!!
